### DISCUSSION TITLE
How could I open in new window/tab/buf when jump to define(gD)

### DISCUSSION THREAD

**Discussion by x2c3z4** [adoyle-h/one.nvim#20] [Q&A] [answered]
When using 'jump to define', the new file always replace the original file buffer.

**Comment by adoyle-h** [comment, marked as answer]
Just type `gd` to open float window to query the definition of the symbol under the cursor.

By the way, in float window, you can type `<C-w>t` to turn current window into a new tab.

Please see [keymaps.md](https://github.com/adoyle-h/one.nvim/blob/master/doc/usage/keymaps.md#lsp-and-diagnostic).
